### PR TITLE
metronome class

### PR DESCRIPTION
**Pull request by kensipe**
Created metronome class wrapper similar to marathon

### CODE REVIEW

**Comment by tamarrow** [comment]
thanks @kensipe. Can we share methods with the Marathon class? At least the RpcClient?

This code is not being used right now. Should be integrated with the job main class: cli/dcoscli/job/main.py

**Comment by kensipe** [comment]
@tamarrow tests locally... ready to go (after CI tests)

**Comment by kensipe** [comment]
@tamarrow looks like CCM failed... otherwise good

**Comment by kensipe** [comment]
rebased

File: dcos/rpcclient.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        :type args: [object]
+        :param kwargs: kwargs to pass to `method_fn`
+        :type kwargs: dict
+        :returns: `method_fn` return value
+        :rtype: requests.Response
+        """
+
+        url = self._base_url + path.lstrip('/')
+
+        if 'timeout' not in kwargs:
```
Comment (tamarrow) [inline dcos/rpcclient.py]: this isn't needed, its set in http.request

File: dcos/rpcclient.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
+import json
```
Comment (tamarrow) [inline dcos/rpcclient.py]: 👍

Comment (tamarrow) [inline dcos/rpcclient.py]: (to this file)

File: cli/tests/data/dcos.toml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [core]
```
Comment (tamarrow) [inline cli/tests/data/dcos.toml]: this file should be reverted

**Comment by kensipe** [comment]
@tamarrow rebased

**Comment by kensipe** [comment]
@tamarrow rebased again, reverted toml file (sorry about that) and pep8 and integration checked.

**Comment by tamarrow** [comment]
thanks @kensipe! Will merge once tests pass

**Pull request merged**